tools/sklearn/model_validation.xml

```diff
@@ -0,0 +1,221 @@
+<tool id="sklearn_model_validation" name="Model Validation" version="@VERSION@">
+    <description>evaluates estimator performance by cross-validation</description>
+    <macros>
+        <import>main_macros.xml</import>
+    </macros>
+    <expand macro="python_requirements"/>
+    <expand macro="macro_stdio"/>
+    <version_command>echo "@VERSION@"</version_command>
+    <command>
+        <![CDATA[
+        cat '$sklearn_model_validation' &&
+        python '$sklearn_model_validation' '$inputs'
+        ]]>
+    </command>
+    <configfiles>
+        <inputs name="inputs" />
+        <configfile name="sklearn_model_validation">
+            <![CDATA[
+import sys
+import json
+import pandas
+import pickle
+import numpy as np
+import sklearn.model_selection
+from sklearn import svm, linear_model, ensemble
+
+import logging
+
+logging.basicConfig(format='%(asctime)s,%(msecs)d %(levelname)-8s [%(filename)s:%(lineno)d] %(message)s',
```

Comment:
Why do you need to change the defaults? And should this go to stdout instead? So the Galaxy user can see it in the end?

Reply:
You mean the logger?

Reply:
Yes.

Reply:
That's for debugging purpose. Will remove those lines finally. You don't a tool to track your code?

Reply:
I use logging, I was just asking why you do modify the defaults and why you are not printing to stdout, this way you would also see it in Galaxy.

Reply:
Sorry to misunderstand you. Seems that printing to stdout is a simple and clean way. Thanks for pointing that out.